data/filters/templates/google-search-cleanup.yaml

```diff
@@ -63,10 +63,11 @@ template: |
   www.google.*###footcnt > #fbarcnt
   {{/if}}
   {{#if question-answer-box}}
-  www.google.*###rhs div[jsdata]:has([class*="wholepage-card"]):nth-of-type(5)
+  www.google.*###rhs #kp-wp-tab-overview div[data-attrid*="qa"]
```

Comment:
`*="qa"` (these two letters anywhere in the string), might be too prone to false-positives, what do you think about using `$=` instead, to match on the suffix?

Reply:
`$=" qa" would be the best, as not only ensure it is a word (because of the extra space before the q - although not completely a word), but also that it is at the end of the string.

Reply:
I also change the other 2 rules and made `*` a `$`.

Reply:
Great, thanks!